hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/web/TestWebHDFS.java

```diff
@@ -2255,6 +2256,35 @@ public void testFileLinkStatus() throws Exception {
     }
   }
 
+  @Test
+  public void testFsStatus() throws Exception {
+    final Configuration conf = WebHdfsTestUtil.createConf();
+    try {
+      cluster = new MiniDFSCluster.Builder(conf)
+          .numDataNodes(1)
+          .build();
+      cluster.waitActive();
+
+      final WebHdfsFileSystem webHdfs =
+          WebHdfsTestUtil.getWebHdfsFileSystem(conf,
+              WebHdfsConstants.WEBHDFS_SCHEME);
+
+      final String path = "/foo";
+      OutputStream os = webHdfs.create(new Path(path));
+      os.write(new byte[1024]);
+
+      FsStatus fsStatus = webHdfs.getStatus(new Path("/"));
+      Assert.assertNotNull(fsStatus);
+
+      //used, free and capacity are non-negative longs
+      Assert.assertTrue(fsStatus.getUsed() >= 0);
+      Assert.assertTrue(fsStatus.getRemaining() >= 0);
+      Assert.assertTrue(fsStatus.getCapacity() >= 0);
```

Comment:
there is already a static import for these. No need of Assert. prefix.

Rather than just asserting they aren't 0, can you get the values from DistributedFileSystem and validate that they are same

Reply:
Thank you for your valuable suggestion. I greatly appreciate it and will promptly make the necessary changes to the code!